pkgs/by-name/li/librime-lua/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  unstableGitUpdater,
+  lua,
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "librime-lua";
+  version = "0-unstable-2024-05-19";
+
+  src = fetchFromGitHub {
+    owner = "hchunhui";
+    repo = "librime-lua";
+    rev = "7be6974b6d81c116bba39f6707dc640f6636fa4e";
+    hash = "sha256-jsrnAFE99d0U0LdddTL7G1p416qJfSNR935TZFH3Swk=";
+  };
+
+  propagatedBuildInputs = [ lua ];
+
+  installPhase = ''
+    mkdir $out
+    cp --archive --verbose src/ $out
+    install --mode=644 --verbose --target-directory=$out CMakeLists.txt  LICENSE  README.md
+  '';
+
+  passthru.updateScript = unstableGitUpdater { hardcodeZeroVersion = true; };
+
+  meta = {
+    description = "Extending RIME with Lua scripts";
+    homepage = "https://github.com/hchunhui/librime-lua";
+    license = lib.licenses.bsd3;
+    maintainers = with lib.maintainers; [ linj ];
```

Comment:
Since you have maintained your own `librime-lua` for a while, would you like to become a maintainer?  @zendo @xddxdd

Reply:
Sure, feel free to add me as a maintainer.